packages/trader/src/Modules/Trading/Containers/trade-chart.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useRef, useEffect } from 'react';
```

Comment:
this is breaking convention for deriv-app repo. use React.useRef and React.useEffect instead